cli/tests/test_parallelize_command.rs

```diff
@@ -241,9 +242,19 @@ fn test_parallelize_failure_disconnected_target_commits() {
     ◉  000000000000
     "###);
 
-    insta::assert_snapshot!(test_env.jj_cmd_failure(
-        &workspace_path, &["parallelize", "description(1)", "description(3)"]),@r###"
-    Error: Cannot parallelize since the target revisions are not connected.
+    let (stdout, stderr) = test_env.jj_cmd_ok(
+        &workspace_path,
+        &["parallelize", "description(1)", "description(3)"],
+    );
+    insta::assert_snapshot!(stdout, @"");
+    insta::assert_snapshot!(stderr, @r###"
+    Nothing changed.
+    "###);
+    insta::assert_snapshot!(get_log_output(&test_env, &workspace_path), @r###"
```

Comment:
I'm not sure if I understand why this is an ok result since 3 is not parallel with respect to 1 after the command runs.

I think I do understand how this happens though. Since 2 is not part of the target set, it retains 1 as a parent. For the same reason 3 retains 2 as a parent. And so there are no changes. I guess this is a result that seems surprising at first, but makes sense and is totally correct if you understand how the command *really* works.

Maybe you can add a comment to the test with the explanation above?

Reply:
Right. I should update the help text to clarify that the command effectively says that the inputs are independent among themselves, but that external dependencies are preserved.

Reply:
Done.